tests/test_routes.py

```diff
@@ -195,8 +195,8 @@ def test_delete_products_from_wishlist(self):
     resp = self.app.get("/wishlists/1/products")
     self.assertEqual(resp.status_code, 405)
 
-  def test_delete_a_product_from_wishlist(self):
```

Comment:
These route tests should remain the same. You should add a test in `test_product` for your new function `delete_by_id ` though.